package.json

```diff
@@ -38,43 +39,37 @@
     "react-calendar": "^3.3.1",
     "react-paginate": "^7.1.5",
     "react-range": "^1.8.11",
-    "react-tooltip-lite": "npm:@carforyou/react-tooltip-lite@1.12.1",
     "tailwindcss-gradients": "^2.0.0",
     "use-deep-compare-effect": "^1.4.0"
   },
   "optionalDependencies": {
     "fsevents": "^2.0.0"
   },
   "devDependencies": {
-    "@babel/core": "7.16.0",
     "@carforyou/assets": "2.1.2",
     "@carforyou/eslint-config": "4.0.0",
     "@carforyou/pika-plugin-build-components": "1.2.4",
     "@pika/pack": "0.5.0",
     "@pika/plugin-copy-assets": "0.9.2",
     "@pika/plugin-standard-pkg": "0.9.2",
     "@pika/plugin-ts-standard-pkg": "0.9.2",
-    "@storybook/addon-actions": "6.3.8",
-    "@storybook/addon-essentials": "6.3.8",
-    "@storybook/addon-links": "6.3.8",
+    "@storybook/addon-actions": "6.4.9",
+    "@storybook/addon-essentials": "6.4.9",
+    "@storybook/addon-links": "6.4.9",
     "@storybook/addon-postcss": "2.0.0",
-    "@storybook/react": "6.3.8",
+    "@storybook/react": "6.4.9",
     "@storybook/storybook-deployer": "2.8.10",
-    "@testing-library/react": "12.1.2",
     "@testing-library/jest-dom": "5.16.1",
+    "@testing-library/react": "12.1.2",
     "@testing-library/user-event": "13.5.0",
     "@types/jest": "27.0.3",
-    "@types/node": "14.18.0",
+    "@types/node": "16.11.13",
     "@types/react": "16.14.21",
     "@types/react-paginate": "7.1.1",
     "autoprefixer": "10.4.0",
-    "babel-loader": "8.2.3",
     "bundlesize": "0.18.1",
     "eslint": "8.4.1",
-    "eslint-plugin-react-hooks": "4.3.0",
     "jest": "27.3.1",
-    "marked": "2.1.3",
-    "marked-terminal": "4.1.1",
```

Comment:
I believe those were added as workarounds for something. we don't use them anywhere explicitly